rasa/nlu/test.py

```diff
@@ -398,6 +398,33 @@ def evaluate_response_selections(
     }
 
 
+def add_confused_intents_to_report(
+    report: Dict, target_intents: Iterable[Any], predicted_intents: Iterable[Any]
+) -> Dict:
+
+    from sklearn.metrics import confusion_matrix
```

Comment:
let's follow the convention to not directly import functions from other modules https://www.notion.so/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e

Reply:
didn't know that, thanks!